### ISSUE TITLE
Performance discrepancy in some maps.

### ISSUE THREAD

**Issue by Henry668** [lich14/CDS#8]
Hello! 
Note that the provided config files do not contain `3s_vs_5z` and `5m_vs_6m`, whose results are included in the paper. Could you provide the config of these 2 maps?

**Comment by Henry668** [comment]
Hi!
I ran `3s_vs_5z` using the hyperparameter settings provided in the appendix of the paper but found the performance does not match the one in the paper. Additionally, the results of `corridor` and `MMM2` also show large discrepancy compared to the ones in the paper. 
I followed the instruction in https://github.com/lich14/CDS/tree/main/CDS_SMAC, and for the latter two maps, I used the default settings in this repo by running
`python3 src/main.py --config=qplex_qatten_sc2 --env-config=sc2_corridor` and `python3 src/main.py --config=qplex_qatten_sc2 --env-config=sc2_MMM2`. 
Is there anything important that I have missed when running the code?

**Comment by lich14** [comment]
First of all thanks for your interest in our work
If you use the latest code, using the default settings should be no problem, because all the code has been tested repeatedly.

1. MMM2: It has been tested by other people, such as https://github.com/lich14/CDS/issues/6.
2. Corridor: This environment is the only one in SMAC that needs to decay the intrinsic reward function weights. An inappropriate reward function decay can degrade performance. I have tested the hyper-parameter 'anneal_rate' in {0.5, 1, 2, 3, 4, 5}  with five random seeds and found that 3 or 5 achieves stable performance. If you are interested, you can try to adjust this parameter.
3. 3s_vs_5z: I haven't paid much attention to this environment, and our algorithm really has no advantage on these hard maps. If the difference between our approach and QPLEX is very large, a possible problem is whether epsilon_anneal_time is set correctly. On the hard maps, this parameter should be 50000.

If there are still problems, I have updated my pip requirement list, but it includes sth that is not related to this code.
